src/stores/app/index.js

```diff
@@ -165,19 +162,24 @@ export default class AppStore extends Store {
       delete params.noLoading;
     }
 
-    const result = await this.request.get(
-      'apps',
-      assign(defaultParams, params)
-    );
-
+    const result = await this.request.get('apps', params);
     const apps = get(result, 'app_set', []);
```

Comment:
If this method tranfer params is same the store parameter like search_word, owner, the store define variable will cover the transfer parameter。

Reply:
user defined params have higher priority